### PR TITLE
Add /etc/service entries to fix Postfix startup errors

### PR DESCRIPTION
**Pull request by bagasme**
This PR will fix issue #2752.

I haven't try this guide locally, but I guessed that the error lies after configuring Postfix **and** before going through Dovecot configuration.

Sorry if the commit message title looks truncated and split.

### CODE REVIEW

**Comment by nmelehan** [comment]
Deploy preview for *nostalgic-ptolemy-b01ab8* ready!

Built with commit 4ee7720288dc890fffd397975d80d1c5539153a9

https://deploy-preview-2771--nostalgic-ptolemy-b01ab8.netlify.com

**Comment by bagasme** [comment]
Please do not merge until I fix commit message title.

**Comment by bagasme** [comment]
OK, I'd force-pushed with fixed commit message, have a CI test running.

File: docs/email/postfix/email-with-postfix-dovecot-and-mysql/index.md [hunk 1]
```diff
@@ -489,6 +489,15 @@ smtps     inet  n       -       -       -       -       smtpd
 
         sudo systemctl restart postfix
 
+    If you get `fatal: 0.0.0.0:smtps: Servname not supported for ai_socktype`
+error after restarting Postfix, add following service entries to
+`/etc/services`:
```
Comment (bagasme) [inline docs/email/postfix/email-with-postfix-dovecot-and-mysql/index.md]: I split this paragraph into multiple lines, instead of single line per paragraph

**Pull request merged**